lib/plugins.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var _ = require('lodash');
+var prefix = 'gemini-';
+
+module.exports = {
+    load: function(gemini, options) {
+        var enabledPlugins = _.reject(_.pairs(options.plugins), function(p) {
```

Comment:
But I don't like converting object to array and referring to name and option by index very much. Maybe, it will be more clear if we'll use: `_.pick(plugins, function(plugin) { return plugin !== false }` here and `_.forEach(enabledPlugins, function(opts, name) {..})` on 11th line.